src/ButteredBread.sol

```diff
@@ -5,8 +5,9 @@ import {OwnableUpgradeable} from "openzeppelin-contracts-upgradeable/contracts/a
 import {ERC20VotesUpgradeable} from
     "openzeppelin-contracts-upgradeable/contracts/token/ERC20/extensions/ERC20VotesUpgradeable.sol";
 import {IERC20} from "openzeppelin-contracts/contracts/token/ERC20/IERC20.sol";
+
 import {IButteredBread} from "src/interfaces/IButteredBread.sol";
-import {IERC20Votes} from "src/interfaces/IERC20Votes.sol";
+import {IBreadToken} from "src/interfaces/IBreadToken.sol";
```

Comment:
So in the latest commit of the Bread token submodule there is an interface for the bread contract, but it does not include the ERC20Votes aspects of it. Unfortunately, OZ does not provide an interface for ERC20Votes so we must make it ourselves. However, because there is already an interface for the Bread token contract this renaming introduces ambiguity and confusion...

Reply:
Can you link to where there is already a Bread token interface? I don't see it anywhere. The intention here was to reduce confusion by naming the interface to be consistent with the way that it was being used. Will change it back if it's doing the opposite.